games/api/serializers.py

```diff
@@ -45,6 +45,7 @@ class Meta:
             "date",
             "table_name",
             "table_extra_notes",
+            "report_notes",
```

Comment:
This does not fully cover requirement on who can see the report. 

Anyone with access to this endpoint can pull the data and read the notes. It should be returned as a null or empty string, if the requesting user is not a DM. So we need a backend to check access to the field. This can be easily covered with `MethodField`.

Reply:
Thanks for the info. I added check in the serializer.